### PR TITLE
Blacklists rubocop Style/StringLiterals.

### PR DESCRIPTION
**Pull request by ashercodeorg**
Also alphabetizes blacklist.

### CODE REVIEW

**Comment by ashercodeorg** [comment]
CC @wjordan

File: .rubocop.yml [hunk 1]
```diff
@@ -58,17 +74,16 @@ Style/IfInsideElse:
...
+# and apathy to the rule to suggest a negative cost-benefit of enabling the
+# rule.
+Style/SpaceAroundEqualsInParameterDefault:
   Enabled: false
 
-Metrics/LineLength:
+# As of July 2016, there is enough disagreement in our codebase, on our team,
+# and apathy to the rule to suggest a negative cost-benefit of enabling the
+# rule.
+Style/StringLiterals:
```
Comment (joshlory) [inline .rubocop.yml:77]: No chance of ever enabling this rule? 😄

**Comment by joshlory** [comment]
In general I favor enforcing more of the style rules and configuring the IDE so you don't have to think about it.  But I can also understand concerns around having too many rules around things we don't care about.

I'm not sure why `Style/SpaceAroundEqualsInParameterDefault` is special (why is it more contentious than [`Style/MethodCallParentheses`](https://github.com/code-dot-org/code-dot-org/pull/9235), for example).

**Comment by ashercodeorg** [comment]
Definitely, I agree with you @joshlory. The "specialness" of `Style/SpaceAroundEqualsInParameterDefault` may simply be the result of my having opened the conversation.

File: .rubocop.yml (hunk 1 above)
Comment (ashercodeorg) [inline .rubocop.yml:77]: Honestly, I'd like to see the rule enabled at some point, there is definite benefit to knowing the string is a vanilla string versus a complex string at a glance. Especially if we could reenable the rule without messing up git blame.

Comment (ashercodeorg) [inline .rubocop.yml:77]: For now, though, I don't think this rule belongs in the todo. E.g., if this were the last rule remaining, I'm not sure we would want to fix it (my bar for moving it to this file).

**Comment by ashercodeorg** [comment]
@joshlory, is the consensus to keep Style/StringLiterals in the blacklist but not Style/SpaceAroundEqualsInParameterDefault?

**Comment by joshlory** [comment]
I'm not sure.  I'd like to hear from @wjordan on whether `rake install:hooks` is working with his process, since he's the main one that had pushback.  I think the rest of the team is either ambivalent or in favor of enabling more lint rules.

**Comment by joshlory** [comment]
Ping @wjordan.

**Comment by ashercodeorg** [comment]
Ping @wjordan. Unless further discussion ensues, I'll modify this PR to only blacklist the Style/StringLiterals rule.

**Comment by ashercodeorg** [comment]
@joshlory PTAL

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -266,11 +266,11 @@ Style/RescueModifier:
...
-# Offense count: 197
-# Cop supports --auto-correct.
-# Configuration parameters: EnforcedStyle, SupportedStyles.
-# SupportedStyles: space, no_space
-Style/SpaceAroundEqualsInParameterDefault:
+# Offense count: 197		
+# Cop supports --auto-correct.		
+# Configuration parameters: EnforcedStyle, SupportedStyles.		
+# SupportedStyles: space, no_space		
+Style/SpaceAroundEqualsInParameterDefault:
```
Comment (joshlory) [inline .rubocop_todo.yml]: Trailing spaces?

Comment (ashercodeorg) [inline .rubocop_todo.yml]: Yep. Sorry. Fixing.

File: .rubocop.yml [hunk 3]
```diff
@@ -37,38 +37,47 @@ Style/Semicolon:
...
+Style/IfInsideElse:
   Enabled: false
 
-Style/AsciiComments:
+Style/RedundantReturn:
   Enabled: false
+
+# As of July 2016, there is enough disagreement in our codebase, on our team,
+# and apathy to the rule to suggest a negative cost-benefit of enabling the
+# rule.
```
Comment (joshlory) [inline .rubocop.yml]: FWIW I think the existing comment covers the current status:

``` yml
# Listed below are rules we don't plan to enable in the foreseeable future.
# Add rules we plan to fix (but are currently failing) to .rubocop_todo.yml.
```

Historical context can be found in these PRs.

File: .rubocop.yml [hunk 4]
```diff
@@ -37,38 +37,47 @@ Style/Semicolon:
 Style/WordArray:
   MinSize: 5
 
-# Listed below are rules we don't plan to enable in the foreseeable future.
-# Add rules we plan to fix (but are currently failing) to .rubocop_todo.yml.
+# END CODE.ORG OVERRIDES
 
-Style/BarePercentLiterals:
+# BEGIN BLACKLIST
```
Comment (joshlory) [inline .rubocop.yml]: Same comment as below (existing comment is more detailed).

**Pull request merged**